.github/workflows/utils/sql-proxy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash -ex
+
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Shared Cloud SQL Proxy setup
+# Presumes the following variables in ci-setup.json:
+#  * CLOUD_SQL_CONNECTION_NAME - the project:region:instance of a Cloud SQL instance.
+#  * UNIX_SOCKET_DIR - a local directory to set the proxy to (default tmp/cloudsql)
+#
+# Note: in GitHub Actions environments, `/cloudsql` is not valid.
```

Comment:
Is this something we should validate against or does the script always does this anyways, and it's only for us to be aware?

Reply:
See above with default value; it's also used as a helper in the `INSTANCE_UNIX_SOCKET` value, which must be prefixed with the location of the folder where the socket is setup to work.